.travis.yml

```diff
@@ -45,7 +45,13 @@ jobs:
       script: make build
       os: osx
     - stage: test
-      script: true
+      script:
+        - go test -v $(go list ./... | grep -vE "vendor|cli|utils")
+        # Allow the cli tests to fail, as some of them rely on external tooling,
+        # such as debugfs.
```

Comment:
I don't think this is a good idea.

IMO, the CLI tests are actually the only ones to test for Mac OS, as the CLI tool is exactly what we want to support on this platform. So we shouldn't just ignore all of them and, instead, find a smarter way to skip (via `testing.skip()`) only the problematic tests.

What do you think? Can you convince me of the opposite?

Reply:
I agree, but it will be a time vs effort decision. If we decide to do the fine-grained solution, this will have to be manually tweaked by everyone, every time a new test is added on a mac.
But I guess you are right. 
I'll start cherry-excluding tests.

Reply:
Give it a try! Thanks :+1: